### PR TITLE
[Android] Add all Activities and xml to act all of func

### PR DESCRIPTION
**Pull request by Joojihwan**
[Edit] NNStreamerActivity : Add connection of views with layout and Edit onClick to act each buttons
[Add] PreviewActivity, activity_preview.xml : Get bitmap Image from NNStreamerActivity and Show the Image. Can save or retry.
[Add] SettingActivity, activity_setting.xml : Set the conditions to send to NNStreamerActivity with spinner, editText

Every layout is not landscape.
Can open own gallery with NNStreamerActvitiy's button
Can capture the surfaceView with countdown
Can save the captured image in PreviewActivity in 'autocapture' folder

Signed-off-by: Joojihwan <wngustnf@naver.com>

### CODE REVIEW

File: android/example_app/nnstreamer-ssd-autocapture/AndroidManifest.xml [hunk 1]
```diff
@@ -15,19 +18,22 @@
...
         android:theme="@android:style/Theme.NoTitleBar.Fullscreen"
         tools:ignore="GoogleAppIndexingWarning">
+        <activity android:name=".PreviewActivity"></activity>
+        <activity android:name=".SettingActivity" />
         <activity
             android:name=".NNStreamerActivity"
-            android:screenOrientation="landscape"
-            android:label="@string/app_name">
+            android:label="@string/app_name"
+            android:screenOrientation="portrait"> <!-- Change Orientation from landscape to portrait-->
```
Comment (park-sang-hyun) [inline android/example_app/nnstreamer-ssd-autocapture/AndroidManifest.xml:31]:
```suggestion
            android:screenOrientation="portrait">
```

Comment (park-sang-hyun) [reply android/example_app/nnstreamer-ssd-autocapture/AndroidManifest.xml]: I'd like to remove the comment.

**Review by park-sang-hyun — CHANGES_REQUESTED** [review]
I'd like to remove the comment.

File: android/example_app/nnstreamer-ssd-autocapture/AndroidManifest.xml [hunk 2]
```diff
@@ -28,12 +28,12 @@
         <activity
             android:name=".NNStreamerActivity"
             android:label="@string/app_name"
-            android:screenOrientation="portrait"> <!-- Change Orientation from landscape to portrait-->
+            android:screenOrientation="portrait">
```
Comment (Joojihwan) [inline android/example_app/nnstreamer-ssd-autocapture/AndroidManifest.xml:31]: Occured a problem about sign-off

**Comment by macaron10** [comment]
LGTM

**Comment by Soyeong-k-1230** [comment]
LGTM

**Comment by park-sang-hyun** [comment]
LGTM!

**Comment by JeongYeonUk** [comment]
LGTM

File: android/example_app/nnstreamer-ssd-autocapture/res/layout/activity_preview.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="1"></LinearLayout>
+
+    </LinearLayout>
+
+
+</LinearLayout>
```
Comment (gichan-jang) [inline android/example_app/nnstreamer-ssd-autocapture/res/layout/activity_preview.xml:65]: Add new line here.

Comment (gichan-jang) [reply android/example_app/nnstreamer-ssd-autocapture/res/layout/activity_preview.xml:65]: Check other files also.

**Comment by zhoonit** [comment]
Please keep newline changes in one commit either with `git rebase -i` the fix commit or `git reset --soft HEAD~1 && git commit --amend` and do `git push -f`

**Review by anyj0527 — APPROVED** [review]
LGTM

**Pull request merged**